notebooks/rendered/python/ex_3.ipynb

```diff
@@ -421,14 +403,16 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "def should_hit(player_total, dealer_total, player_aces):\n",
+    "def should_hit(dealer_total, player_total, player_low_aces, player_high_aces):\n",
```

Comment:
I saw in the forums that some people were confused by the phrase dealer_total.

They thought it reflected the total of the dealer's two cards, as opposed to the card showing (the visible total).  

I would consider renaming this to `dealer_card` or `dealer_visible_total`.  

Of those, I prefer `dealer_card` even though someone could conceivably think it could take values like `K` or `Q`.

Reply:
Hm, this is tricky. I do agree that there's some potential for confusion here, but I'm a +0 on those alternatives, just because I'm not convinced they're less confusing. But if you want to go ahead and make the change here or in the micro-challenge, that's fine with me.

Reply:
I called it `dealer_card_val` in the challenge.  

It'll be interesting to see what questions we get, or whether the code people post in forums suggests an implicit misunderstanding.